### PR TITLE
Update snarkvm dependency

### PR DESCRIPTION
**Pull request by apruden2008**
Update snarkvm revs that are pinned in Cargo.toml in setup1 contributor to account for updates in snarkVM private key generation logic

### CODE REVIEW

**Comment by ibaryshnikov** [comment]
@apruden2008 can you also update it in setup1-cli-tools? Other than that, looks good

**Comment by weikengchen** [comment]
I am propagating the updates to remaining repos and fixing here and there---the new version changes the name of one of the repos.

File: phase1/tests/marlin.rs [hunk 1]
```diff
@@ -8,19 +8,23 @@ mod test {
...
+    use blake2::Blake2s;
     use itertools::Itertools;
+    use snarkvm_marlin::FiatShamirChaChaRng;
+    use snarkvm_polycommit::sonic_pc::SonicKZG10;
     use std::{collections::BTreeMap, ops::MulAssign};
 
     #[test]
     fn test_marlin_posw_bls12_377() {
-        let powers = 19usize;
+        let powers = 18usize;
```
Comment (weikengchen) [inline phase1/tests/marlin.rs:27]: Note: It seems that 18 is sufficient for the circuit being tested.

File: phase1/tests/marlin.rs [hunk 2]
```diff
@@ -100,7 +113,7 @@ mod test {
             .mine(&subroots, difficulty_target, &mut rand::thread_rng(), std::u32::MAX)
             .unwrap();
 
-        assert_eq!(proof.len(), 972); // NOTE: Marlin proofs use compressed serialization
+        assert_eq!(proof.len(), 771); // NOTE: Marlin proofs use compressed serialization
```
Comment (weikengchen) [inline phase1/tests/marlin.rs:116]: Note: The proof size has been reduced.

File: phase1/tests/marlin.rs [hunk 3]
```diff
@@ -69,9 +73,16 @@ mod test {
...
+        let mut neg_shift_powers_of_h = BTreeMap::new();
+        let mut degree_bounds = vec![];
+        tau_powers_g2[2..].iter().enumerate().skip(1).for_each(|(i, p)| {
+            degree_bounds.push((1 << i) - 2);
+            neg_shift_powers_of_h.insert((1 << i) - 2, (*p).clone());
+            shift_powers_of_g.insert(
+                (1 << i) - 2,
+                tau_powers_g1[parameters.powers_length - 1 - (1 << i) + 2].clone(),
+            );
         });
```
Comment (weikengchen) [inline phase1/tests/marlin.rs:86]: This is to supply additional fields needed for `UniversalParams`.

File: phase1/tests/marlin.rs [hunk 4]
```diff
@@ -143,11 +156,17 @@ mod test {
...
+        <Bls12_377 as PairingEngine>::Fq,
+        SonicKZG10<Bls12_377>,
+        FiatShamirChaChaRng<<Bls12_377 as PairingEngine>::Fr, <Bls12_377 as PairingEngine>::Fq, Blake2s>,
+        snarkvm_marlin::marlin::MarlinTestnet1Mode,
+    >;
 
     #[test]
     fn test_marlin_sonic_pc() {
-        let powers = 16usize;
+        let powers = 15usize;
```
Comment (weikengchen) [inline phase1/tests/marlin.rs:169]: Same. It seems that 15 is sufficient.

File: setup2/src/cli/new.rs [hunk 5]
```diff
@@ -65,17 +65,12 @@ pub fn new(opt: &NewOpts) -> anyhow::Result<()> {
...
 
-        let private_program_input = Execution::<<Testnet2Parameters as Parameters>::ProgramSNARK> {
-            verifying_key: noop_program_snark_parameters.1.clone(),
-            proof: program_snark_proof,
-        };
+        let noop_circuit = dpc
+            .noop_program
+            .find_circuit_by_index(0)
+            .ok_or(DPCError::MissingNoopCircuit)?;
+        let private_program_input = dpc.noop_program.execute_blank(noop_circuit.circuit_id())?;
```
Comment (weikengchen) [inline setup2/src/cli/new.rs:73]: The `dpc.noop_program` now has a new API to generate a dummy proof.

**Comment by weikengchen** [comment]
This PR is ready for review. Note that although tests in `phase1_coordinator` are passing, there are some error messages showing up. Please decide whether they are problematic.

**Comment by apruden2008** [comment]
thanks @weikengchen.

**Review by ibaryshnikov — APPROVED** [review]

**Pull request merged**